lesson-5/Alam - Lesson 5/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  font-family: "Work Sans", sans-serif;
+}
+
+.container {
+  background-color: #F7F2F2;
+  padding: 0px;
+  height: 485px;
+  width: 275px;
+}
+
+.text {
+  text-align: left;
+  padding: 15px 30px;
+}
+
+div p {
+  font-size: 14px;
+  margin-bottom: 30px;
+}
+
+div a {
+  font-size: 14px;
+  padding: 8px 85px;
```

Comment:
In case this were to be a more responsive element (changes with the size of the page), setting an explicit padding like this might be restrictive. We can use `display: block;` here to make sure that no matter what the size of the containing element (in this case the `div`), it will always take up all of the horizontal space available.